Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -573,18 +574,15 @@ def _invalidate(self, which=None):
                         item.setData(None, Qt.DisplayRole)
                         item.setData(None, Qt.ToolTipRole)
 
-        self.apply_button.setEnabled(True)
+        if self.auto_apply:
```

Comment:
Skip this `if`.

`gui.auto_commit` replaces the original commit method, which checks this condition.